README.md

```diff
@@ -127,8 +127,11 @@ Example:
 
 ### Pull Requests
 
-- Pull Request title should be the same as commit message
+- Pull Request title should be the same as the commit message
```

Comment:
Please check this point in your PR

Reply:
Fixed. Thank you.

Reply:
The requirement has been removed.

Reply:
I don't get it, requirement removed? Requirement is still in place

Reply:
Never mind, now everything is according to requirement 😄